### ISSUE TITLE
why use the totalStride?

### ISSUE THREAD

**Issue by brealisty** [bertinetto/siamese-fc#57]
I don't understand the code 
`disp_instanceInput = disp_instanceFinal * p.totalStride / p.responseUp;`

why *p.totalStride.  cuz the stride of conv?

**Comment by jvlmdr** [comment]
Yes. Usually there will be several layers in the network with stride > 1. This means that displacements in the response map need to be multiplied by the overall stride of the network.

**Comment by guwenning** [comment]
@jvlmdr  Hello, I don't quite understand the 8 coordinates in ground_truth. I used a trace with a number of coordinates of 4. I found that the coordinates of the first frame did not completely wrap the target on the target. If I use my own sequence of images, I should enter the first What should be the coordinate format of a frame of picture?
